src/ssl.c

```diff
@@ -19,6 +19,7 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
  */
 
+#include <stdint.h>
```

Comment:
Not all platforms will have <stdint.h> available.  This should be protected by a define like "#ifndef NO_STDINT".  Since it is also only used in the OpenSSL compatibility layer, it should be inside a check for OPENSSL_EXTRA.